internal/restapi/browse/frontend/package.json

```diff
@@ -12,8 +12,8 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^29.5.5",
     "@types/node": "^16.11.62",
-    "@types/react": "^18.0.21",
-    "@types/react-dom": "^18.2.8",
+    "@types/react": "^18.2.23",
+    "@types/react-dom": "^8.2.8",
```

Comment:
Was there a reason to change this one from **1**8.2.8 to 8.2.8 you just an accident?

Reply:
By accident.

Reply:
Ok that's what I suspected, can you revert the version and then I'll approve and merge.
Thanks for this!